WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasMiniTalent_Cata.lua

```diff
@@ -0,0 +1,327 @@
+if not WeakAuras.IsLibsOK() then
+  return
```

Comment:
Also minimizebutton:SetPoint("RIGHT", frame.CloseButton, "LEFT", WeakAuras.IsClassicEraOrWrath() and  10 or 0, 0) in OptionsFrame.lua